go-controller/pkg/util/net_linux.go

```diff
@@ -170,15 +170,30 @@ func LinkNeighExists(link netlink.Link, neighIP net.IP, neighMAC net.HardwareAdd
 	return false, nil
 }
 
-func DeleteConntrack(ip string) {
+func DeleteConntrack(ip string, ports []int32) {
 	ipAddress := net.ParseIP(ip)
 	if ipAddress == nil {
 		klog.Errorf("Value \"%s\" passed to DeleteConntrack is not an IP address", ipAddress)
 		return
 	}
 
 	filter := &netlink.ConntrackFilter{}
-	err := filter.AddIP(netlink.ConntrackReplyAnyIP, ipAddress)
+	err := filter.AddProtocol(17)
```

Comment:
@JacobTanenbaum can you add a comment to the effect that 17 = UDP?

Reply:
SCTP will need to flush the conntrack entries too

Reply:
done